biosimulations/apps/dispatch-service/src/app/app.controller.ts

```diff
@@ -1,3 +1,4 @@
+import { urls } from './../../../../libs/config/common/src/lib/urls';
```

Comment:
Cant use this relative import. Needs to be changed to the @biosimulations/ pattern

Reply:
Sure, will do it.